keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,454 @@
+            residual block in the model.
+        stackwise_stride: list of ints, stride length for each inverted
+            residual block in the model.
+        filters: integer, the number of filters for the convolution layer.
+        include_rescaling: bool, whether to rescale the inputs. If set to True,
+            inputs will be passed through a `Rescaling(scale=1 / 255)`
+            layer.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e., output of `layers.Input()`)
+            to use as image input for the model.
+        alpha: float, controls the width of the network. This is known as the
+            depth multiplier in the MobileNetV3 paper, but the name is kept for
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input
+            units to drop during training, defaults to 0.2.
+
+    Examples:
+    ```python
+    input_data = tf.ones(shape=(8, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = MobileNetV3Backbone(
+        stackwise_expansion=[1, 72.0 / 16, 88.0 / 24, 4, 6, 6, 3, 3, 6, 6, 6],
+        stackwise_filters=[16, 24, 24, 40, 40, 40, 48, 48, 96, 96, 96],
+        stackwise_stride=[2, 2, 1, 2, 1, 1, 1, 1, 2, 1, 1],
+        filters=1024,
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        stackwise_expansion,
+        stackwise_filters,
+        stackwise_kernel_size,
+        stackwise_stride,
+        stackwise_se_ratio,
+        stackwise_activation,
+        filters,
+        include_rescaling,
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        alpha=1.0,
+        dropout_rate=0.2,
+        **kwargs,
+    ):
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+        x = inputs
+
+        if include_rescaling:
+            x = layers.Rescaling(scale=1 / 255)(x)
+
+        x = layers.Conv2D(
+            16,
+            kernel_size=3,
+            strides=(2, 2),
+            padding="same",
+            use_bias=False,
+            name="Conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=CHANNEL_AXIS,
+            epsilon=BN_EPSILON,
+            momentum=BN_MOMENTUM,
+            name="Conv/BatchNorm",
+        )(x)
+        x = apply_hard_swish(x)
+
+        pyramid_level_inputs = {}
+        for stack_index in range(len(stackwise_filters)):
+            if stackwise_activation[stack_index] == "hard_swish":
```

Comment:
Let's handle this inside `apply_inverted_res_block` as suggested above.

We also don't need to reduce flexibility by assuming it's hard swish OR relu. Just process `"hard_swish"` inside that function or pass along the string as is.

Reply:
Unless I'm terribly mistaken, the activations are applied independently and not as a part of another layer.

Reply:
You are just passing the activation to `apply_inverted_res_block`. You can handle the string parsing there 😄 

Again, don't make "relu" the default, just handle the "hard_swish" string

Reply:
> Unless I'm terribly mistaken, the activations are applied independently and not as a part of another layer.

I was referring to the `apply_inverted_res_block ` here, that if we don't handle the "relu" string as well in a similar way that we're approaching for the "hard_swish", it won't work AFAIK. I'll make a commit soon.